packages/tooling/__tests__/lib/parameters-to-json-schema.test.js

```diff
@@ -335,43 +358,7 @@ describe('request bodies', () => {
   });
 
   describe('$ref support', () => {
-    it('should work for top-level request body $ref', () => {
```

Comment:
I removed this test case because pulling a `$ref` for a `requestBody` that has no `content` or a mime type isn't actually validated OAS.